ospf6d/ospf6_message.c

```diff
@@ -888,12 +890,13 @@ static void ospf6_lsreq_recv(struct in6_addr *src, struct in6_addr *dst,
 		/* Find database copy */
 		lsa = ospf6_lsdb_lookup(e->type, e->id, e->adv_router, lsdb);
 		if (lsa == NULL) {
-			if (IS_OSPF6_DEBUG_MESSAGE(oh->type, RECV)) {
-				zlog_debug(
-					"Can't find requested [%s Id:%pI4 Adv:%pI4]",
-					ospf6_lstype_name(e->type), &e->id,
-					&e->adv_router);
-			}
+			char id[16], adv_router[16];
+			inet_ntop(AF_INET, &e->id, id, sizeof(id));
+			inet_ntop(AF_INET, &e->adv_router, adv_router,
+				  sizeof(adv_router));
+			zlog_warn(
+				"Can't find requested lsa [%s Id:%s Adv:%s] send badLSReq",
```

Comment:
Why remove the `%pI4` format specifiers?

Reply:
sorry about that, my mistake fixed

Reply:
my mistake, fixed